fecfile_validate_python/src/fecfile_validate/validate.py

```diff
@@ -62,7 +64,7 @@ def parse_schema_error(error):
     return ValidationError(error.message, path)
 
 
-def validate(schema_name, form_data, fields_to_validate=None):
+def validate(schema_name, form_data, fields_to_validate=None, version="8.5"):
```

Comment:
@toddlees @sasha-dresden do we want to fail loudly here if they don't pass in a schema? (same with get_schema function).  Originally I was thinking we could default to 8.5, but I feel like that could lead to obscure bugs if we forget to update the default version one day or forget it's being used.